test/python/circuit/library/test_evolved_op_ansatz.py

```diff
@@ -17,7 +17,8 @@
 from qiskit.circuit import QuantumCircuit
 from qiskit.quantum_info import SparsePauliOp, Operator, Pauli
 
-from qiskit.circuit.library import EvolvedOperatorAnsatz, HamiltonianGate
```

Comment:
this line should still work, right?

Reply:
It should I made the change so it imports EvolvedOperatorAnsatz the same way other n_local classes are imported to match the formatting more closely. Either way of importing should import the new class in n_local package.